album/pokemon-scarlet-violet-super-music-collection.yaml

```diff
@@ -0,0 +1,1900 @@
+Album: 'Pokémon Scarlet & Pokémon Violet + The Hidden Treasure of Area Zero: Super Music Collection'
+Directory: pokemon-scarlet-violet-super-music-collection
+Artists:
+- Pokémon
+- GAME FREAK
+Date: February 27, 2024 00:00:02
+## https://vgmdb.net/album/134816
+Date Added: 
+URLs:
+- https://www.youtube.com/playlist?list=PLjjzf1NYDNrlrtLn41YRK1uPjQZl8GnvX
+Cover Artists:
+- Toshiyuki Takahashi
+- Hiromi Yasuda
+- Momoko Fukuda
+- 上野愛恵 ##can't find who this is lol
+Cover Art File Extension: png
+Color: '#f8ce00'
+Groups:
+- Toby Fox
+- Beyond
+Wallpaper Artists:
+- GAME FREAK
+Wallpaper Style: |-
+    image-rendering: pixelated;
+    image-rendering: crisp-edges;
+Commentary: |-
+    <i>Toby Fox:</i> (UNDERTALE / DELTARUNE NEWSLETTER - ISSUE 1 (Winter Season), excerpt)
```

Comment:
This should probably be a link (also for the other excerpts from the newsletter).